lib/src/core/message/guild_emoji.dart

```diff
@@ -1,40 +1,66 @@
-import 'package:nyxx/src/nyxx.dart';
-import 'package:nyxx/src/core/snowflake.dart';
-import 'package:nyxx/src/core/snowflake_entity.dart';
-import 'package:nyxx/src/core/guild/guild.dart';
-import 'package:nyxx/src/core/guild/role.dart';
-import 'package:nyxx/src/core/message/emoji.dart';
+import 'package:nyxx/nyxx.dart';
 import 'package:nyxx/src/internal/cache/cacheable.dart';
-import 'package:nyxx/src/typedefs.dart';
 
 abstract class IBaseGuildEmoji implements SnowflakeEntity, IEmoji {
   /// True if emoji is partial.
   bool get isPartial;
 
-  /// Returns cdn url to emoji
-  String get cdnUrl;
+  /// The name of the emoji.
+  String get name;
+
+  /// Whether this emoji is animated.
+  bool get animated;
 
   /// Creates partial emoji from given String or Snowflake.
-  factory IBaseGuildEmoji.fromId(Snowflake id) => GuildEmojiPartial(id);
+  factory IBaseGuildEmoji.fromId(Snowflake id) => GuildEmojiPartial({"id": id});
```

Comment:
Can it be returned as string? Because its inconsistent with BaseGuildEmoji since it takes snowflake as string